### DISCUSSION TITLE
ITRON's ERT gas meter 100G DLN data to be captured using CC1111 usb dongle from TI

### DISCUSSION THREAD

**Discussion by rohitjindal786** [bemasher/rtlamr#252] [General]
Hi ALL

I am new to this ERT Technology. We have a requirement to capture the data of Itron's the ERT Gas meter 100g DLN using CC1111 Usb dongle from Texas Instruments.

CC1111 transceiver Usb dongle : from TI capture some fix formatted packets under frequency ranges from 868/915 specifically ISM Bands, using some good tools provided like Smart RF Packet Sniffer, Smart RF Studio by TI. 
[Datasheet https://www.ti.com/lit/gpn/cc1110-cc111](https://www.ti.com/lit/gpn/cc1110-cc1111). 
CC1111 understanding :
1 Fixed format packet support.  We can set the how many byte preamble of packet will be that can be 2 byte /3/4/5 bytes but we cant set what will be  value of preamble.
2 We can just set 16bit SYNC0/SYNC1 sync word but we cant set that packets 30/32 bit sync word but we can declare that packet can have 30/32 bit sync word but there is option to set only 16bit sync word.
3Variable /fixed length data length support
4 Manchestor encode/decode support but not well understandable

Issue
1 We are trying to capture the ERT Gas meter 100g DLN Data using CC1111 since long but we did not succeed due to now well understanding of Gas meter due to limited or no open support available.
2Gas meter has very few bits for sync and preamble ie only 21 Bits for SCM packet but for cc1111 dongle we require atleast 2bytes for SYNC WORD and 4BYTE for Preamble . For best case Sync/preamble should be 4byte each. 
3ERT Technology wiki page suggest that data is manchestor encoded but we are not still sure . I am not able to capture that in RTLAMR code here also if data being captured by RTL Dongle is manchester encoded data that need tobe decoded.
4 If any body can provide any similar meter data that they are able to capture using TI Dongle 
    or Raw data from rtlamr code  just before decoding it to required fields that would be great help
Please guide.

**Comment by rohitjindal786** [comment]
Hi ALL,
Please guide

**Reply by rohitjindal786** [reply]
any body pls guide

**Reply by rohitjindal786** [reply]
@bemasher pls guide we are struggling since long for capturing data with cc1111 dongle from ERT Gas meter.

At least if you can confirm if this will work with CC1111 dongle or not?

**Comment by bemasher** [comment]
Yes, you can receive and decode ERT-flavored messages on a CC1111 dongle.

This repository and it's discussions are meant for discussing rtlamr, not design help for other projects. Locking.
